### PR TITLE
Add packaging for OTI xlsx

### PR DESCRIPTION
**Pull request by alexrichey**

### CODE REVIEW

**Comment by alexrichey** [comment]
> If you update python reqs and recompile, mypy should run fine here

🤦 yes, forgot entirely about the compile step. Been a while since I updated a dependency!

File: python/constraints.txt [hunk 1]
```diff
@@ -380,6 +380,8 @@ numpy==2.1.0
     #   streamlit
 openpyxl==3.1.5
     # via -r python/requirements.in
+openpyxl-stubs==0.1.25
```
Comment (alexrichey) [inline python/constraints.txt:383]: Adding these in manually until the breaking change with the reqs is resolved.

**Review by damonmcc — APPROVED** [review]
so cool! was gonna say let's add excel generation to Template DB [here](https://github.com/NYCPlanning/data-engineering/blob/main/products/template/build_scripts/export.py#L96), but happy to wait and align testing the of excel and PDF generation later

**Comment by alexrichey** [comment]
> so cool! was gonna say let's add excel generation to Template DB [here](https://github.com/NYCPlanning/data-engineering/blob/main/products/template/build_scripts/export.py#L96), but happy to wait and align testing of the excel and PDF generation later

@damonmcc yes! great idea. Giving that a go now.

**Comment by alexrichey** [comment]
@damonmcc added another commit to generate the xlsxs - last commit if you want to glance over it

**Pull request merged**